lib/src/main/kotlin/WeatherSensorService/Public/CustomStringConvertible.kt

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
"CustomStringConvertible" is a Swift thing. This file should be called something like DoubleExtensions.kt.